integration-tests/src/test/java/org/wildfly/prospero/it/cli/CloneTest.java

```diff
@@ -0,0 +1,170 @@
+        try (InstallationMetadata meta = metadata) {
+            ChannelManifest channelManifest = meta.getManifest();
+            ProsperoConfig prosperoConfig = meta.getProsperoConfig();
+            ProvisioningConfig provisionedConfig = meta.getGalleonProvisioningConfig();
+            // check channelManifest
+            assertEquals(ChannelManifestMapper.CURRENT_SCHEMA_VERSION, channelManifest.getSchemaVersion());
+            assertNull(channelManifest.getDescription());
+            assertEquals(164, channelManifest.getStreams().size());
+            assertTrue(channelManifest.getStreams().stream()
+              .anyMatch(m -> m.getGroupId().equals("io.undertow")
+                && m.getArtifactId().equals("undertow-core")
+                && m.getVersion().equals("2.2.17.Final")));
+            assertTrue(channelManifest.getStreams().stream()
+              .anyMatch(m -> m.getGroupId().equals("org.jboss.xnio")
+                && m.getArtifactId().equals("xnio-nio")
+                && m.getVersion().equals("3.8.7.Final")));
+            assertTrue(channelManifest.getStreams().stream()
+              .anyMatch(m -> m.getGroupId().equals("org.wildfly.core")
+                && m.getArtifactId().equals("wildfly-cli")
+                && m.getVersion().equals("19.0.0.Beta11")));
+
+            // check prosperoConfig
+            assertEquals(1, prosperoConfig.getChannels().size());
+            Channel channel = prosperoConfig.getChannels().get(0);
+            assertEquals("channel-0", channel.getName());
+            assertEquals(3, channel.getRepositories().size());
+            channel.getRepositories().stream().anyMatch(p -> p.getId().equals("maven-central"));
+            channel.getRepositories().stream().anyMatch(p -> p.getId().equals("nexus"));
+            channel.getRepositories().stream().anyMatch(p -> p.getId().equals("maven-redhat-ga"));
+
+            // check provisionedConfig
+            assertTrue(provisionedConfig.hasFeaturePackDeps());
+            assertEquals(1, provisionedConfig.getFeaturePackDeps().size());
+            FeaturePackConfig featurePackConfig = provisionedConfig.getFeaturePackDeps().iterator().next();
+            assertEquals("wildfly-core", featurePackConfig.getLocation().getProducerName());
+        }
+    }
+
+    @Test
+    public void testImport() throws Exception {
+        mockInstallationMeta(targetDir);
+        new InstallationExportAction(targetDir).export(exportPath.toString());
+        ExecutionUtils.prosperoExecution(CliConstants.Commands.CLONE, CliConstants.Commands.IMPORT,
+            CliConstants.DIR, importDir.toString(),
+            CliConstants.ARG_PATH, exportPath.toString()
+          )
+          .execute()
+          .assertReturnCode(ReturnCodes.SUCCESS);
+        checkMetaData(new InstallationMetadata(importDir));
+    }
+
+    @Test
+    public void testImportOverrideRepositories() throws Exception {
+        mockInstallationMeta(targetDir);
```

Comment:
Could we modify the `${targetDir}/.installation/manifest.yaml`  so that the `org.wildfly.core:wildfly-cli` stream has version `19.0.0.Beta12-SNAPSHOT`? Then we can check that the temporary repository is indeed used?